src/amo/components/SearchSortLink/index.js

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+
+import { makeQueryString } from 'core/api';
+import { convertFiltersToQueryParams } from 'core/searchUtils';
+import Link from 'amo/components/Link';
+
+import './SearchSortLink.scss';
+
+
+export function queryString(filters) {
+  return makeQueryString(convertFiltersToQueryParams(filters));
+}
+
+export default class SearchSortLink extends React.Component {
```

Comment:
I'm cool with that. I much prefer components in their own file; the consistency helps me and I think definitely helps newbies. Determining "what is big enough for its own file and what should just be another component declared as a `const` in the same file" is way more overhead than "New... File" 😉